app/models/form526_submission.rb

```diff
@@ -373,6 +376,8 @@ def submit_form_8940
 
   def submit_flashes
     user = User.find(user_uuid)
+    # Note that the User record is stored in Redis temporarily -- `User.redis_namespace_ttl`
+    # If this method runs after the TTL, then the flashes will not be applied -- a possible bug.
```

Comment:
Does this happen? Seems like if a user is active on the site, they'd have a user in Redis and if they weren't on the site, they wouldn't need a flash message.

Reply:
If the job fails and retries, it could take days for the job to eventually succeed. By that time, the `User` record could be expired.

Reply:
AFAIK, it doesn't currently happen, but it is possible since the current TTL is 30 minutes and Sidekiq jobs can retry 8 times with longer delays with each subsequent retry.